app/src/main/res/values/history_strings.xml

```diff
@@ -43,7 +43,7 @@
     <string name="transaction_details_pending_transfer_description">The transaction will be completed in a few seconds</string>
 
     <string name="transaction_details_share">Share</string>
-    <string name="transaction_details_explorer">Explorer</string>
+    <string name="transaction_details_explorer">SolScan</string>
```

Comment:
for referral reward transaction details, I used `SOLScan` naming (history_strings.xml `transaction_details_referral_button_second`) because it's been added to figma

Reply:
thanks!